src/apis/contracts_api.ts

```diff
@@ -416,6 +416,14 @@ export class ContractsAPI {
         return this.loadERC20TokenAsync(tokenAddress, transactionOptions);
     }
 
+    public async doesTokenCorrespondToSymbol(
+        principalToken: string,
```

Comment:
Should this name be more generic, like `tokenAddress`?

Reply:
+1

Reply:
Agreed